assets/javascript/game.js

```diff
@@ -1,219 +1,95 @@
 $(document).ready(function(){
 
-			var wins = 0;
-			var losses = 0;
-			
-			// This is the sum of the guesses added together. 
-			var counter = 0;
+  var audioElement1 = document.createElement('audio');
+  audioElement1.setAttribute('src', 'https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.WAV');
 
-			var crystal1 = 0;
-			var crystal2 = 0;
-			var crystal3 = 0;
-			var crystal4 = 0;
+  var audioElement2 = document.createElement('audio');
+  audioElement2.setAttribute('src', 'https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.mp3');
 
-			var audioElement1 = document.createElement("audio");
-      		audioElement1.setAttribute("src", "https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.WAV");
+  var song = document.createElement('audio');
+  song.setAttribute('src', 'https://s3-us-west-1.amazonaws.com/week4sounds/sounds/song.mp3');
 
-      		var audioElement2 = document.createElement("audio");
-      		audioElement2.setAttribute("src", "https://s3-us-west-1.amazonaws.com/week4sounds/sounds/register.mp3");
+  var wins = 0;
+  var losses = 0;
 
-      		var song = document.createElement("audio");
-      		song.setAttribute("src", "https://s3-us-west-1.amazonaws.com/week4sounds/sounds/song.mp3");
+  var counter;
+  var numberGuess;
 
+  var crystal1 = $( '#crystal1_span' );
```

Comment:
Changed these to reference jQuery objects so they could store a `crystal-value` data property.